README.md

```diff
@@ -46,20 +46,20 @@ Note that the rule is namespaced with `plugin/`.
 
 Like so:
 
-```js
+```json
```

Comment:
I know that `.stylelintrc` contains a json and not js, but the syntax highlight in our README is better with a js.

Reply:
Agreed!